pygments/lexers/procfile.py

```diff
@@ -0,0 +1,40 @@
+"""
+    pygments.lexers.procfile
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for Procfile file format.
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.lexer import RegexLexer, bygroups
+from pygments.token import Name, Number, String, Text, Punctuation
+
+__all__ = ["ProcfileLexer"]
+
+
+class ProcfileLexer(RegexLexer):
+    """
+    Lexer for Procfile file format.
+
+    The format is used to run processus on Heroku or is used by Foreman or
+    Honcho tools.
+    For more information about the definition of the format, see:
+    https://devcenter.heroku.com/articles/procfile#procfile-format
+    """
```

Comment:
Please add a `.. versionadded:: 2.10` line to the comment.

Reply:
I added it in the docstring.